src/Symfony/Component/Notifier/Bridge/Sipgate/README.md

```diff
@@ -0,0 +1,24 @@
+Sipgate Notifier
+===============
+
+Provides [Sipgate](https://www.sipgate.de) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+SIPGATE_DSN=sipgate://TOKENID:TOKEN@default?senderId=SENDERID
+```
+
+where:
+ - `TOKENID` is your Sipgate API Token ID
+ - `TOKEN` is your Sipgate API TOKEN
+ - `SENDERID` is your Sipgate device id (e.g. s1)
```

Comment:
```suggestion
 - `SENDERID` is your Sipgate device ID (e.g. s1)
```

Reply:
If it is the device ID, why not name it `DEVICE_ID`?

Reply:
Actually its called _smsId_ in the official sipgate documentation (https://api.sipgate.com/v2/doc#/sessions/sendWebSms), I just found it confusing and therefore changed it to senderId, as it references to the ID of the sending account.